data-products-dp0-2/index.rst

```diff
@@ -98,112 +95,68 @@ Coadd images are divided into ``tracts`` (a spherical convex polygon) and tracts
-and to use its WCS and methods associated with the image to extract a cutout image zoomed in on a region of interest.
-
+**Difference Images**:
+A PVI which has had a template image subtracted from it.
+Template images are built from images obtained the previous year.
+Any source detected in a difference image represents the *time-variable* flux component of the astrophysical object.
 
 .. _DP0-2-Data-Products-DPDD-Catalogs:
 
 Catalogs
 --------
 
-Source detection, measurement, and characterization have been run on both the PVIs and coadds to generate catalog data for DP0.2 (see also :ref:`Data-Processing-Overview-Coadded-Catalogs`).
+Source detection, measurement, and characterization have been run on the PVIs, coadds, and difference images to generate catalog data.
 Catalog data are accessible with the :ref:`Data-Access-Analysis-Tools-TAP` via the Portal or Notebook Aspect, and with the Butler via the Notebook Aspect.
-Although this will not be the case for the Operations-era data releases, for DP0.2 the TAP and Butler table data are not named or organized the same way.
-Here we distinguish between the TAP- and Butler-accessible catalog data products.
-**The recommended catalog interface for DP0.2 is the TAP service.**
 
-**Schema:**
-A table's "schema" refers to the column names, units, and descriptions of the tabulated data.
-Links to full or curated versions of the table schema (curated meaning limited to columns that will be of most use to most DP0 delegates) are provided in the tables below.
-A `DP0.2 schema browser <https://dm.lsst.org/sdm_schemas/browser/dp01.html>`_ is also available, thanks to the Rubin Data Management team.
+**DP0.2 Table Schema:**
+The column names, units, and descriptions of the DP0.2 catalogs listed in the table below are all available via the `DP0.2 schema browser <https://dm.lsst.org/sdm_schemas/browser/dp02.html>`_.
+
+.. list-table:: Catalog data available for DP0.2.
+   :widths: 100 100 390
+   :header-rows: 1
+
+   * - TAP Name
+     - Butler Name
+     - Description
+   * - Object
+     - objectTable
+     - Astrometric and photometric measurements for objects detected in coadded images (990 columns).
+   * - Source
+     - sourceTable
+     - Astrometric and photometric measurements for sources detected in the individual PVIs (143 columns).
+   * - ForcedSource
+     - forcedSourceTable
+     - Forced photometry on the individual PVIs at the locations of all detected objects and sources (38 columns).
+   * - DiaObject
+     - diaObjectTable_tract
+     - Derived summary parameters for DiaSources associated by sky location, including lighcurve statistics (137 columns).
+   * - DiaSource
+     - diaSourceTable_tract
+     - Astrometric and photometric measurements for sources detected in the difference images (66 columns).
+   * - ForcedSourceOnDiaObject
+     - forcedSourceOnDiaObjectTable_tract
```

Comment:
There is also `forcedSourceOnDiaObjectTable`, which is per-patch.

Reply:
yes there is. changed to reporting that in table as main table